docs/index.css

```diff
@@ -46,3 +46,55 @@ code {
 .Demo ul {
   margin: var(--space-large);
 }
+
+.Footer {
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+  flex: none;
+  flex-direction: row;
+  box-sizing: border-box;
+  z-index: var(--z-index-sidebar);
```

Comment:
Do you know why this is needed?

Reply:
Leftover from when I was trying to avoid `position: fixed`. Good catch!